Source/WebKit/Shared/RemoteLayerTree/RemoteLayerBackingStore.mm

```diff
@@ -317,7 +328,7 @@ static bool hasValue(const ImageBufferBackendHandle& backendHandle)
         return false;
     }
 
-    if (m_layer->owner()->platformCALayerDelegatesDisplay(m_layer)) {
+    if (m_layer->owner() && m_layer->owner()->platformCALayerDelegatesDisplay(m_layer)) {
```

Comment:
Is there any chance that `m_layer->owner()` could be nulled between `m_layer->owner()` and `m_layer->owner()->`? Would it be better to place the result of `m_layer->owner()` in a temporary variable?

Reply:
I don't think so, but I agree with the pattern :)

I might actually remove this change, as the other changes mean that we never end up here without an owner. Probably better to just consider it invalid to run this code without an owner, rather than silently returning a different answer.